go.mod

```
@@ -1,27 +1,27 @@
 module github.com/golioth/xk6-coap
 
-go 1.20
+go 1.19
```

Let's keep this at `1.20` 👍🏻 
```suggestion
go 1.20
```